app/static/css/_results.scss

```diff
@@ -38,6 +36,7 @@
 }
 
 .result__doi {
+    height: 1rem;
```

Comment:
Putting a height on this container means that search results that do not have a DOI displayed have the same size margin between the result title and the top of the result container as the ones that do. It was a weird little thing to look at.